ams/ams-server/src/main/java/com/netease/arctic/ams/server/terminal/TerminalManager.java

```diff
@@ -91,34 +93,35 @@ public String executeScript(String terminalId, String catalog, String script) {
     CatalogMeta catalogMeta = optCatalogMeta.get();
     TableMetaStore metaStore = getCatalogTableMetaStore(catalogMeta);
     String sessionId = getSessionId(terminalId, metaStore);
+    String catalogType = CatalogUtil.isIcebergCatalog(catalog) ? "iceberg" : "arctic";
+    Configuration configuration = new Configuration();
+    configuration.setInteger(SessionConfigOptions.FETCH_SIZE, resultLimits);
+    configuration.set(SessionConfigOptions.CATALOGS, Lists.newArrayList(catalog));
+    configuration.set(SessionConfigOptions.catalogConnector(catalog), catalogType);
+    configuration.set(SessionConfigOptions.CATALOG_URL_BASE, String.format(
+        "thrift://%s:%d/",
+        ArcticMetaStore.conf.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST),
+        ArcticMetaStore.conf.getInteger(ArcticMetaStoreConf.THRIFT_BIND_PORT)));
```

Comment:
` configuration.set(SessionConfigOptions.CATALOG_URL_BASE, String.format(
        "thrift://%s:%d/",
        ArcticMetaStore.conf.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST),
        ArcticMetaStore.conf.getInteger(ArcticMetaStoreConf.THRIFT_BIND_PORT)));`
shall we consider AMS ha mode??